src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Attributes/JsonObjectCreationHandlingAttribute.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization.Metadata;
+
+namespace System.Text.Json.Serialization;
+
+/// <summary>
+/// When placed on a field or property, indicates if member will replaced or populated.
+/// When placed on a type with <see cref="JsonObjectCreationHandling.Populate"/> indicates that all members which are capable of being populated will be.
+/// </summary>
+/// <remarks>
+/// For attributes placed on fields and properties with default resolvers this will be reflected with <see cref="JsonPropertyInfo.ObjectCreationHandling"/>.
+/// For attributes placed on types this will be reflected with <see cref="JsonTypeInfo.PreferredPropertyObjectCreationHandling"/>.
+/// Note the attribute corresponds only to the preferred values of properties.
+/// Only property type is taken into consideration. For example if property is of type
+/// <see cref="IEnumerable{T}"/> but it is assigned <see cref="List{T}"/> it will not be populated
+/// because <see cref="IEnumerable{T}"/> is not capable of populating.
+/// Additionally value types require a setter to be populated in which case populating means
```

Comment:
Remove additionally and split into a separate paragraph
```suggestion
///
/// Value types require a setter to be populated in which case populating means
```

Reply:
will be included in the next iteration